File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/AboutGoods/EntryMetaData.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.AboutGoods;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryMetaData {

    @SerializedName("meta_description")
    @Expose
    private String metaDescription;
    @SerializedName("meta_title")
    @Expose
    private String metaTitle;

    public String getMetaDescription() {
        return metaDescription;
    }

    public void setMetaDescription(String metaDescription) {
        this.metaDescription = metaDescription;
    }

    public String getMetaTitle() {
        return metaTitle;
    }

    public void setMetaTitle(String metaTitle) {
        this.metaTitle = metaTitle;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/aboutgoods/EntryMetaData.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.aboutgoods

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryMetaData {

    @SerializedName("meta_description")
    @Expose
    var metaDescription: String? = null
    @SerializedName("meta_title")
    @Expose
    var metaTitle: String? = null

}
